assets/evaluation_on_cloud/environments/evaluations-built-in/context/requirements.txt

```diff
@@ -2,4 +2,13 @@ azure-ai-evaluation
 openai
 azureml-mlflow
 azure-identity
-azure-ai-ml
\ No newline at end of file
+azure-ai-ml
+azureml-core
```

Comment:
SDK team advised that it is not recommended to combine v1 and v2 SDK. Please use either v1 or v2.

Please also pin the versions and us templates for AzureML SDK dependnencies